google-cloud-clients/google-cloud-storage/src/test/java/com/google/cloud/storage/BucketInfoTest.java

```diff
@@ -83,6 +83,9 @@
   private static final Long RETENTION_EFFECTIVE_TIME = 10L;
   private static final Long RETENTION_PERIOD = 10L;
   private static final Boolean RETENTION_POLICY_IS_LOCKED = false;
+  private static final List<String> LOCATION_TYPES =
+      ImmutableList.of("multi-region", "region", "dual-region");
+  private static final String LOCATION_TYPE = "multi-region";
```

Comment:
Just a curious question: How did you decide upon which should be default LocationType?

Reply:
Does  `BucketInfor.newBuilder()` sets fields with default data?

Reply:
its not default value taken it is used just for unit test you can use any one from "multi-region", "region", "dual-region"